setup.py

```diff
@@ -8,7 +8,7 @@
     "chiapos==1.0.10",  # proof of space
     "clvm==0.9.7",
     "clvm_tools==0.4.5",  # Currying, Program.to, other conveniences
-    "chia_rs==0.1.9",
+    "chia_rs==0.1.10",
```

Comment:
This seems unrelated?

Reply:
I meant to comment on this in the PR. the `Coin` class wasn't declared "sendable" in rust, which means it wasn't allowed to be sent to a different thread. I had to fix that in order to pass in a coin to the `get_puzzle_and_solution_for_coin()` function, with it running in a thread pool